src/PowerShellEditorServices/Services/PowerShellContext/Console/ConsoleReadLine.cs

```diff
@@ -48,8 +48,9 @@ public async Task<SecureString> ReadSecureLineAsync(CancellationToken cancellati
         {
             SecureString secureString = new SecureString();
 
-            int initialPromptRow = await ConsoleProxy.GetCursorTopAsync(cancellationToken);
-            int initialPromptCol = await ConsoleProxy.GetCursorLeftAsync(cancellationToken);
+            // TODO: Are these values used?
+            int initialPromptRow = await ConsoleProxy.GetCursorTopAsync(cancellationToken).ConfigureAwait(false);
+            int initialPromptCol = await ConsoleProxy.GetCursorLeftAsync(cancellationToken).ConfigureAwait(false);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'initialPromptCol'.](https://app.codacy.com/manual/TylerLeonhardt/PowerShellEditorServices/pullRequest?prid=4812399)